wrappers/ios/Sources/Wrapper/Reader/ZXIGTIN.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 KURZ Digital Solutions GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ZXIGTIN : NSObject
+@property(nonatomic, nonnull)NSString *country;
+@property(nonatomic, nonnull)NSString *addOn;
```

Comment:
Question: is `nonnull` a deliberate choice here? I don't know the convention in the ObjectiveC/Swift space regarding `null` vs empty string, just thought that maybe `null` could possibly be the canonical way of expressing "does not exists".